mesh/mesh.go

```diff
@@ -825,3 +826,25 @@ func (msh *Mesh) GetATXs(atxIds []types.ATXID) (map[types.ATXID]*types.Activatio
 	}
 	return atxs, mIds
 }
+
+// GetATXsByCoinbase returns a list of all atx ids corresponding to a coinbase address
+func (msh *Mesh) GetATXsByCoinbase(coinbase types.Address) ([]types.ATXID, error) {
+
+	var atxIds []types.ATXID
+	atxIter := msh.GetAtxIterByCoinbase(coinbase)
+	for atxIter.Next() {
+		var a types.ATXID
+		err := types.BytesToInterface(atxIter.Value(), &a)
+		if err != nil {
+			return nil, err
+		}
+		atxIds = append(atxIds, a)
+	}
+
+	atxIter.Release()
+	if err := atxIter.Error(); err != nil {
```

Comment:
I had that question too! I couldn't find it in the documentation but my educated guess would be any other iterator errors (inc running multiple iterators concurrently) would cause iter.Next() to be false and then the error surfaces when you do iter.Error. Not sure how I can confirm this theory though

Reply:
I think the answer is [here](https://godoc.org/github.com/syndtr/goleveldb/leveldb/iterator#CommonIterator):

>     // Error returns any accumulated error. Exhausting all the key/value pairs
>     // is not considered to be an error.
> ...
> When encounter an error any 'seeks method' will return false and will yield no key/value pairs. The error can be queried by calling the Error method. Calling Release is still necessary.